assets/js/blocks/cart-checkout/checkout-i2/styles/editor.scss

```diff
@@ -27,12 +27,12 @@
 
 body.wc-lock-selected-block--move {
 	.block-editor-block-mover__move-button-container,
-	.block-editor-block-mover,
-	.block-editor-block-settings-menu {
```

Comment:
@mikejolley you added this. Was it necessary? leaving causes a double border in the block toolbar and the whole menu to disappear (including insert before and after).

Reply:
This was for hiding UI elements in older versions.

Reply:
I think it removes the up/down arrows and delete button. The double border is a side effect of this but I think we need it until your change ships.

Reply:
For me, `.block-editor-block-settings-menu` was causing the whole menu to be removed. the arrows are handled by `.block-editor-block-mover`
I will check previous versions, because if we remove the menu, it becomes very hard to add blocks around a certain block using the `Insert Before` and `Insert After` buttons.

Reply:
It does however hide the copy link, which is currently broken - you can copy/paste our form step blocks.

Reply:
We can try to hide the Copy button until we fix it in GB
p1628779688045200-slack-C45SNKV4Z